File: defects-explorer/src/main/java/cn/edu/sustc/cse/sql/defectsexplorer/persistence/PullRequestsRepository.java
package cn.edu.sustc.cse.sql.defectsexplorer.persistence;

import cn.edu.sustc.cse.sql.defectsexplorer.entities.PullRequest;
import org.springframework.data.mongodb.repository.MongoRepository;

public interface PullRequestsRepository extends MongoRepository<PullRequest, Integer> {
}

File: defects-explorer/src/main/kotlin/cn/edu/sustc/cse/sql/defectsexplorer/persistence/PullRequestsRepository.kt
package cn.edu.sustc.cse.sql.defectsexplorer.persistence

import cn.edu.sustc.cse.sql.defectsexplorer.entities.PullRequest
import org.springframework.data.mongodb.repository.MongoRepository

interface PullRequestsRepository : MongoRepository<PullRequest, Int>
